client/src/components/DropdownTraveller.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import "../styles/DropdownTraveller.scss";
+
+function DropdownTraveller() {
+  const [selectedOption, setSelectedOption] = useState("");
+
+  const handelSelectChange = (event) => {
+    setSelectedOption(event.target.value);
+  };
+
+  return (
+    <>
+      <label className="dropDownTravellerLabel" htmlFor="selectNumberTraveller">
+        Nombre de passagers :
+      </label>
+      <select
+        name="travellers"
+        id="selectNumberTraveller"
+        value={selectedOption}
+        onChange={handelSelectChange}
+        className="dropDownTravellerSelect"
+      >
+        {[...Array(10)].map((_, index) => (
+          <option key={Array.id} value={index + 1}>
```

Comment:
What is the value of this id?

Reply:
finaly i've created an array of 10 options and give the id of the array in map key